package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@apollo/space-kit",
-  "version": "3.4.0",
+  "version": "3.4.1-canary.0e047a6.0",
```

Comment:
Can we bump this to a minor version?

Reply:
Ewwwww this should not have been in the PR!

Reply:
Removed; the version bump happens _after_ merge; this was an accident 😃